File: app/src/main/java/com/example/TeethHealth/VisualElements.java
package com.example.TeethHealth;

import android.view.View;

public class VisualElements {

    public static void SetVisibles(int visibles, View[] _visualElements) {
        for (int i = 0; i < _visualElements.length; i++){
            _visualElements[i].setVisibility(visibles);
        }
    }
}

File: app/src/main/java/com/example/TeethHealth/ImageFilters/VisualElements.kt
package com.example.TeethHealth.ImageFilters

import android.view.View

object VisualElements {
    fun SetVisibles(visibles: Int, _visualElements: Array<View?>) {
        for (i in _visualElements.indices) {
            _visualElements[i]?.visibility = visibles
        }
    }
}